src/web/themes/custom/bcgov_career/templates/content/node--career-profile--full.html.twig

```diff
@@ -113,12 +113,20 @@
             <div class="data">
               {% if node.field_median_salary.value is not empty %}
                 <p class="data-item"><span class="f-d">Annual Salary</span><span class="s-d">${{ node.field_median_salary.value|number_format(0, '.', ',') }}</span></p>
+              {% else %}
+                <p class="data-item"><span class="f-d">Annual Salary</span><span class="s-d">Not available</span></p>
               {% endif %}
               {% if node.field_education_level.value is not empty %}
-                <p class="data-item"><span class="f-d">Education Level</span><span class="s-d">{{ node.field_education_level.entity.name.value }}</span></p>
+                <p class="data-item"><span class="f-d">Training, Education, Experience and Responsibilities</span><span class="s-d">{{ node.field_education_level.entity.name.value }}</span></p>
               {% endif %}
               {% if node.field_job_openings.value is not empty %}
-                <p class="data-item"><span class="f-d">Job Openings</br><span>{{ node.field_opening_from_to.value }}</span></span><span class="s-d">{{ node.field_job_openings.value|number_format(-4, ',') }}</span></p>
+                {% if node.field_job_openings.value < 0 %}
```

Comment:
I'm not crazy about duplicating the template for this line just to cap to zero. Can't we do a `max(0, value)` somewhere above this line?

Reply:
updated